frontend/src/app/data/calculation/calculation.effects.ts

```diff
@@ -37,9 +37,12 @@ export class CalculationEffects {
   )));
 
   deleteCalculation$ = createEffect(() => this.actions$.pipe(
```

Comment:
Note: the action has no "reducer" of its own updating local state, instead calculations and scenarios are reloaded from the backend on success, in turn triggering their respective reducers for populating the "Scenarios" and "Previous calculations" views.